docs/quickstart/crud-operations.rst

```diff
@@ -85,13 +87,103 @@ fetch it via the REST API:
           "name": "Doe, John"
         }
       ],
-      "id": 1,
+      "id": "1",
       "title": "Some title"
     },
     "revision": 0,
-    "updated": "2018-05-23T13:28:19.426213+00:00"
+    "updated": "2019-11-19T10:20:45.728060+00:00"
   }
 
+.. _upload-a-file:
+
+Upload a file
+-------------
+
+You can upload a file to a record.
+
+.. code-block:: shell
+
+  # Create an example file
+  $ echo 'example' > example.txt
+  # Upload a file named example.txt to the record with pid of 1
+  $ curl -k -X PUT http://localhost:5000/api/records/1/files/example.txt \
+     -H "Content-Type: application/octet-stream" \
+     --data-binary @example.txt
+
+  {
+     "delete_marker": false,
+     "key": "example.txt",
+     "updated": "2019-11-19T10:26:32.780466",
+     "size": 8,
+     "created": "2019-11-19T10:26:32.769314",
+     "checksum": "md5:ddce269a1e3d054cae349621c198dd52",
+     "links": {
+       "self": "https://localhost:5000/api/records/%3Cinvenio_records_rest.utils.LazyPIDValue%20object%20at%200x1114e9320%3E/files/example.txt?bucket_id=6ed8acbd-72a3-47b7-b070-c7a22f6b6919",
+       "version": "https://localhost:5000/api/records/%3Cinvenio_records_rest.utils.LazyPIDValue%20object%20at%200x1114e9320%3E/files/example.txt?bucket_id=6ed8acbd-72a3-47b7-b070-c7a22f6b6919&versionId=d27bf300-f02e-4ee1-a80c-d17b112d377e",
+       "uploads": "https://localhost:5000/api/records/%3Cinvenio_records_rest.utils.LazyPIDValue%20object%20at%200x1114e9320%3E/files/example.txt?bucket_id=6ed8acbd-72a3-47b7-b070-c7a22f6b6919?uploads"
+     },
+     "version_id": "d27bf300-f02e-4ee1-a80c-d17b112d377e",
+     "mimetype": "text/plain",
+     "tags": {},
+     "is_head": true
+   }
+
+.. _list-files-of-a-record:
+
+List files of a record
+----------------------
+
+Get the list of files for the record.
+
+.. code-block:: shell
+
+  $ curl -k -X GET https://localhost:5000/api/records/1/files?prettyprint=1
+
+  {
+  "contents": [
+    {
+      "delete_marker": false,
+      "key": "example.txt",
+      "updated": "2019-11-19T10:26:32.780466",
+      "size": 8,
+      "created": "2019-11-19T10:26:32.769314",
+      "checksum": "md5:ddce269a1e3d054cae349621c198dd52",
+      "links": {
+        "self": "https://localhost:5000/api/records/%3Cinvenio_records_rest.utils.LazyPIDValue%20object%20at%200x1115ab400%3E/files/example.txt?bucket_id=6ed8acbd-72a3-47b7-b070-c7a22f6b6919",
```

Comment:
Is `invenio_records_rest.utils.LazyPIDValue` supposed to be there?

Reply:
Nope, something was wrong with my Invenio instance.